src/components/tile/tile.style.ts

```diff
@@ -43,7 +43,6 @@ const StyledTile = styled.div<StyledTileProps>`
     border-radius: ${roundness === "default"
       ? "var(--borderRadius100)"
       : "var(--borderRadius200)"};
-    overflow: hidden;
```

Comment:
**suggestion:** When I add this back in locally, the visual issues I've pointed out as `issues` are all fixed.

Reply:
thanks - but unfortunately the `overflow: hidden` is making it so that a Textarea that fills the whole Tile content area (or anything focusable close to the edge when there's very little padding, in theory) gets most of its focus outline chopped off.

I think I can fix the issues you've pointed out (thanks for those) by adding some border-radius to the header and footer - let me have a try anyway 👍

Reply:
yeah agree the approach you're suggesting @robinzigmond is preferable here if possible

Reply:
![image](https://github.com/Sage/carbon/assets/44157880/c049fbcc-9a55-4416-9f58-57babe41c97b)
It also looks like your getting radius on the top corners here which doesn't match the design fyi